app/grandchallenge/evaluation/models.py

```diff
@@ -437,6 +437,10 @@ class Phase(UUIDModel, ViewContentMixin):
         blank=True,
         on_delete=models.SET_NULL,
     )
+    number_of_submissions_limit = models.PositiveSmallIntegerField(
```

Comment:
I think we need to differentiate this from `submissions_limit` more. Maybe `total_number_of_submissions_allowed`? Then maybe add a help text that this is the limit on the total number of submissions for this phase, not per user. Maybe we should also rename `submissions_limit`?

Reply:
Good point, a helptext and renaming it to `total_number_of_submissions_allowed` sounds like a good idea. `submissions_limit` already has a very clear help text, so I think it's not strictly necessary to rename this field, but we could change it to `user_submission_limit`?

Reply:
Sounds find, could even go more verbose with `user_number_of_submissions_allowed_per_period`.

Reply:
`submissions_limit_per_user_per_period` sounds good, for the total limit I prefer `total_number_of_submissions_allowed`

Reply:
Changing `submission_limit` is quite a bit of refactoring. Should we do this in a seperate PR so that this one stays clean?

Reply:
Yes separate PR and then rebase.